### PR TITLE
Adding some usage documentation.

### PR DESCRIPTION
**Pull request by jjmontesl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,145 @@
...
+See the Host OS section below for further information on how to setup your host OS before start.
+
+
+## Usage
+
+The 'build.sh' script included is used to build the containers themselves.
+
+Run the command using:
+
+    ./build.sh 3.2 mono-6.12.0.114 2020-02
```
Comment (akien-mga) [inline README.md:38]: The `2020-02` part is not necessary, there are two forms with either 2 or 4 arguments:
```
  echo "Usage: $0 <godot branch> <mono version> [<mono branch> <mono commit hash>]"
  echo
  echo "Examples: $0 3.1 mono-5.18.1.3"
  echo "        $0 master mono-6.6.0.160 2019-08 bef1e6335812d32f8eab648c0228fc624b9f8357"
```

The form with two arguments relies on a `mono/mono` tag (here `mono-6.12.0.114`) so the branch name is not needed. The form with four arguments is to specify both the branch and the commit (and then the second argument is not a tag but just the name to put in the image version).

File: README.md [hunk 2]
```diff
@@ -1,10 +1,145 @@
...
+    REPOSITORY                 TAG                             IMAGE ID      CREATED       SIZE
+    localhost/godot-ubuntu-64  3.2-mono-6.12.0.114             90aaecfcf17f  3 days ago    1.08 GB
+    localhost/godot-windows    3.2-mono-6.12.0.114             599caf446bd3  3 days Ago    4.42 GB
+    localhost/godot-mono-glue  3.2-mono-6.12.0.114             10542d0d9c58  3 days Ago    1.73 GB
+    localhost/godot-mono       3.2-mono-6.12.0.114             726d09a93466  3 days ago    1.51 GB
+    localhost/godot-export     3.2-mono-6.12.0.114             d61f79d1573d  3 days ago    1.09 GB
+    localhost/godot-fedora     3.2-mono-6.12.0.114             a71b68c67fea  3 days ago    690 MB
+    localhost/fedora           32                              27a979020952  5 weeks ago   208 MB
+    docker.io/i386/ubuntu      trusty                          7d0c7adb414a  5 months ago  196 MB
+    docker.io/library/ubuntu   trusty                          df043b4f0cf1  5 months ago  206 MB
```
Comment (akien-mga) [inline README.md:138]: Here's the full list without the superflous details and listed in build order:
```
REPOSITORY                                       TAG                    SIZE
localhost/godot-fedora                           3.2-mono-6.12.0.114    692 MB
localhost/godot-export                           3.2-mono-6.12.0.114    1.09 GB
localhost/godot-mono                             3.2-mono-6.12.0.114    1.51 GB
localhost/godot-mono-glue                        3.2-mono-6.12.0.114    1.73 GB
localhost/godot-msvc                             3.2-mono-6.12.0.114    11.5 GB
localhost/godot-windows                          3.2-mono-6.12.0.114    4.42 GB
localhost/godot-ubuntu-64                        3.2-mono-6.12.0.114    1.08 GB
localhost/godot-ubuntu-32                        3.2-mono-6.12.0.114    1 GB
localhost/godot-javascript                       3.2-mono-6.12.0.114    4.72 GB
localhost/godot-android                          3.2-mono-6.12.0.114    19.7 GB
localhost/godot-osx                              3.2-mono-6.12.0.114    2.84 GB
localhost/godot-ios                              3.2-mono-6.12.0.114    4.53 GB
```

File: build.sh [hunk 3]
```diff
@@ -20,7 +20,7 @@ fi
 if [ -z "$1" -o -z "$2" ]; then
   echo "Usage: $0 <godot branch> <mono version> [<mono branch> <mono commit hash>]"
   echo
-  echo "Examples: $0 3.1 mono-5.18.1.3"
+  echo "Examples: $0 3.2 mono-6.12.0.114 2020-02"
```
Comment (akien-mga) [inline build.sh:23]:
```suggestion
  echo "Examples: $0 3.2 mono-6.12.0.114"
```
As mentioned above, the branch is not needed.

**Comment by akien-mga** [comment]
Left some comments, looks great!

File: build.sh (hunk 3 above)
Comment (jjmontesl) [reply build.sh]: I have updated as per your comments. Cheers.

**Comment by jjmontesl** [comment]
Updated as per your comments. Cheers!

File: README.md [hunk 4]
```diff
@@ -1,10 +1,143 @@
...
+    localhost/godot-ubuntu-32                        3.2-mono-6.12.0.114    1 GB
+    localhost/godot-javascript                       3.2-mono-6.12.0.114    4.72 GB
+    localhost/godot-android                          3.2-mono-6.12.0.114    19.7 GB
+    localhost/godot-osx                              3.2-mono-6.12.0.114    2.84 GB
+    localhost/godot-ios                              3.2-mono-6.12.0.114    4.53 GB
+
+In addition to this, generating containers will also require some host disk space (around 4.5GB)
+for the downloaded Mono sources and dependencies.
+
+(Note that this list does not show some containers: Android, HTML5, iOS, OSX and MSVC.)
```
Comment (akien-mga) [inline README.md:141]: This note is now outdated. :)

**Comment by jjmontesl** [comment]
I should have reviewed the entire document better. Sorry for that. I don't see anything else now.

**Comment by akien-mga** [comment]
Thanks! That's a great writeup.

**Pull request merged**